custom_components/hilo/__init__.py

```diff
@@ -413,10 +413,10 @@ def set_state(self, entity, state, new_attrs={}, keep_state=False, force=False):
                 LOG.warning(f"Unable to set state because there's no current: {params}")
                 return
             attrs = {}
```

Comment:
Je cherche des poux un peu ici, mais si on pouvait faire comme ça, je crois que ça serait mieux.

```suggestion
            attrs = {
                "last_update": datetime.now()
            }
```

Si c'est compliqué pour toi de changer ça, j'vais merger et fixer un autre fois aussi :-)